samples/Sentry.Samples.Console.Basic/Program.cs

```diff
@@ -88,6 +104,8 @@ async Task ThirdFunction()
         // Simulate doing some work
         await Task.Delay(100);
 
+        SentrySdk.Logger.LogFatal("Crash imminent!", [], static log => log.SetAttribute("suppress", true));
```

Comment:
Odd that we must pass an empty array as a second argument.
There's no way we can set up the overloads in a way that this is not needed?

Reply:
We might be able to have an overload that doesn't take the second `parameters` argument right?